components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/signup/UserSelfRegistrationManager.java

```diff
@@ -643,19 +643,44 @@ public UserRecoveryData introspectUserSelfRegistration(String code, String verif
                                                            String verifiedChannelClaim, Map<String, String> properties)
             throws IdentityRecoveryException {
 
-        return validateSelfRegistrationCode(code,verifiedChannelType,verifiedChannelClaim,properties);
+        return introspectUserSelfRegistration(false, code,verifiedChannelType,verifiedChannelClaim,properties);
+    }
+
+    /**
+     * Introspect the user self registration by validating the confirmation code, sets externally verified claims and
+     * return the details. Does not invalidate the code.
+     *
+     * @param skipExpiredCodeValidation   Skip confirmation code validation against expiration.
```

Comment:
Seems like there is a formatting issue. Shall we align the param descriptions and end all the descriptions with a full stop? Also add the return type info as well?